views/audit/big_query_elt_script_logging.sql

```diff
@@ -0,0 +1,533 @@
+CREATE OR REPLACE VIEW `project_id.dataset_id.view_name` AS (
```

Comment:
give the view a more useful name, one which users would likely keep. e.g. "bq_scripting_logs"